packages/components/src/components/buttons/Toggle.tsx

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
<img width="731" alt="image" src="https://user-images.githubusercontent.com/1206987/161906609-d348e783-02c1-43d1-b9ed-c8ebe999833e.png">

This component is spitting out a warning in form config wizard